schema/meta_schema_language_go.yaml

```diff
@@ -159,7 +159,11 @@ typeSupportStatuses:
   - type: ExperimentalPrometheusMetricExporter
     status: unknown
     notes: ""
-    propertyOverrides: []
+    propertyOverrides:
+      - property: without_scope_info
+        status: unknown
+      - property: without_target_info
+        status: unknown
```

Comment:
#nit: Technically unnecessary since the entire ExperimentalPrometheusMetricExporter has status unknown